changelog.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+0.4.0 (2020-05-12)
+------------------
+
+* Updated to latest Ketting 6 alpha.
+* Both the hook and HoC listen for the `update` event and automatically
+  re-render when they are triggered.
+* The `state` property in the useResource hook is renamed to `resourceState`
+  for consitancy with the HoC.
+* All `body` properties are renamed to `data`.
+
```

Comment:
curious about this, since I have gotten tripped up before with `body` vs `data` in using fetch vs axios for example. Is there an accepted best practice pattern or like, history of this sort of thing?

Reply:
I originally went for `body` to match what `fetch` is doing, but switched this to `data` to match what apollo is doing. Idk if this is the right move, but there is a meaningful difference between the body of a HTTP response, and the data that we extract for it.

In HAL the two are more or less the same thing (except `data` doesnt have `_links` or `_embedded`, but in other formats the 'data' in a response is a much smaller subset from the entire 'body' that we got from the server.

So in some ways, `data` is a subset from `body`.

Ultimately, I don't know what the best name for this thing is. Apollo using the `data` convention is what drove me here because one kinda goal is to get a GraphQL-like experience to rest services.